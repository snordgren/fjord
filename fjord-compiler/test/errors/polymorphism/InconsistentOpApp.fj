module errors.parametricPolymorhpism.InconsistentApply

import TestUtil

(++) : a. a -> a -> a
(++) x y =
  rm x y

five : Int
five = 
  "hello" ++ 5
