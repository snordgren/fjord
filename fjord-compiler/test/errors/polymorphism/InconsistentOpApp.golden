test/errors/polymorphism/InconsistentOpApp.fj:11:10:
   |
11 |   "hello" ++ 5
   |          ^^^^^^
expression has type Int, expected String
